core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/CDCTokenFactory.java

```diff
@@ -0,0 +1,84 @@
+package com.datastax.oss.driver.internal.core.metadata.token;
+
+import com.datastax.oss.driver.api.core.metadata.token.Token;
+import com.datastax.oss.driver.api.core.metadata.token.TokenFactory;
+import com.datastax.oss.driver.api.core.metadata.token.TokenRange;
+import com.datastax.oss.driver.shaded.guava.common.base.Preconditions;
+import java.nio.ByteBuffer;
+import net.jcip.annotations.ThreadSafe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ThreadSafe
+public class CDCTokenFactory implements TokenFactory {
+
+  public static final String PARTITIONER_NAME = "com.scylladb.dht.CDCPartitioner";
+
+  public static final CDCToken MIN_TOKEN = new CDCToken(Long.MIN_VALUE);
+  public static final CDCToken MAX_TOKEN = new CDCToken(Long.MAX_VALUE);
+
```

Comment:
Why are those fields public?

Reply:
No good reason aside from the fact that other (already implemented) TokenFactories also have them public:

https://github.com/scylladb/java-driver/blob/017b27315aabbc813bfda4070f778cbdb37907ec/core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/Murmur3TokenFactory.java#L29-L30

Reply:
ok.